### PR TITLE
Guard jvmtitests newly added header includes to J9ZOS only

### PR DESCRIPTION
**Pull request by Deigue**
Have draft PR ready here @r30shah 
Double checking right now the below, before I remove it from draft:
- full vm open-xl build
- java 8 build (would like more insight on what all platforms we should verify here to minimize surprises later)

### CODE REVIEW

File: runtime/tests/jvmtitests/src/com/ibm/jvmti/tests/decompResolveFrame/decomp003.c [hunk 1]
```diff
@@ -22,7 +22,9 @@
 
 #include "jvmti_test.h"
 #include <string.h>
+#if defined(J9ZOS390)
 #include <strings.h>
+#endif
```
Comment (keithc-ca) [inline runtime/tests/jvmtitests/src/com/ibm/jvmti/tests/decompResolveFrame/decomp003.c:27]: Please repeat the condition here:
```c
#endif /* defined(J9ZOS390) */
```
(in both files)

**Review by keithc-ca — CHANGES_REQUESTED** [review]

File: runtime/tests/jvmtitests/src/com/ibm/jvmti/tests/decompResolveFrame/decomp003.c (hunk 1 above)
Comment (Deigue) [reply runtime/tests/jvmtitests/src/com/ibm/jvmti/tests/decompResolveFrame/decomp003.c]: Done

**Review by keithc-ca — APPROVED** [review]

**Comment by Deigue** [comment]
Updated commit message to match title.

**Comment by keithc-ca** [comment]
Can you mark this as ready for review? Windows builds will be broken until this is merged.

**Pull request merged**